app/Livewire/Installer/PanelInstaller.php

```
@@ -218,4 +225,35 @@ public function createAdminUser(UserCreationService $userCreationService): User
             throw new Halt(trans('installer.exceptions.create_user'));
         }
     }
+
+    public function installEggs(EggImporterService $eggImporterService): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$eggImporterService'. (undefined)

(UnusedFormalParameter)